contrib/rpm/buildah.spec

```diff
@@ -114,35 +114,33 @@ make DESTDIR=%{buildroot} PREFIX=%{_prefix} install install.completions
 - Imagename is required for commit atm, update manpage
 
 * Wed Apr 4 2018 Dan Walsh <dwalsh@redhat.com> 0.16-1
--   Add support for shell
--   Vendor in latest containers/image
--    	 docker-archive generates docker legacy compatible images
--	 Do not create $DiffID subdirectories for layers with no configs
-- 	 Ensure the layer IDs in legacy docker/tarfile metadata are unique
--	 docker-archive: repeated layers are symlinked in the tar file
--	 sysregistries: remove all trailing slashes
--	 Improve docker/* error messages
--	 Fix failure to make auth directory
--	 Create a new slice in Schema1.UpdateLayerInfos
--	 Drop unused storageImageDestination.{image,systemContext}
--	 Load a *storage.Image only once in storageImageSource
--	 Support gzip for docker-archive files
--	 Remove .tar extension from blob and config file names
--	 ostree, src: support copy of compressed layers
--	 ostree: re-pull layer if it misses uncompressed_digest|uncompressed_size
--	 image: fix docker schema v1 -> OCI conversion
--	 Add /etc/containers/certs.d as default certs directory
--  Change image time to locale, add troubleshooting.md, add logo to other mds
--   Allow --cmd parameter to have commands as values
--   Document the mounts.conf file
--   Fix man pages to format correctly
--   buildah from now supports pulling images using the following transports:
--   docker-archive, oci-archive, and dir.
--   If the user overrides the storage driver, the options should be dropped
--   Show Config/Manifest as JSON string in inspect when format is not set
--   Adds feature to pull compressed docker-archive files
-
-
+- Add support for shell
+- Vendor in latest containers/image
+- docker-archive generates docker legacy compatible images
```

Comment:
Can you indent these changes that were due to the vendor please?  lines 119-134.   Otherwise it's tough to tell what came in due to the vendor vs changes directly to Buildah.
Otherwise LGTM